proxy/coco/src/request/waiting_room.rs

```diff
@@ -150,14 +148,14 @@ impl<T, D> WaitingRoom<T, D> {
     ///
     /// If there is no such `urn` then it create a fresh `Request` using the `urn` and `timestamp`
     /// and it will return `None`.
-    pub fn request(&mut self, urn: RadUrn, timestamp: T) -> Option<SomeRequest<T>>
+    pub fn request(&mut self, urn: &RadUrn, timestamp: T) -> Option<SomeRequest<T>>
```

Comment:
Out of curiosity, why change this?

Reply:
It was inconsistent with the other methods. Was there a semantic significance to expect a value instead of a reference?

Reply:
I can't remember :sweat_smile: Thanks for making it more consistent :+1: